xmake.lua

```diff
@@ -81,7 +81,7 @@ local modules = {
 	},
 	Shader = {
 		Deps = {"NazaraCore"},
-		Packages = {"efsw", "fmt", "frozen"},
+		Packages = {"cxxopts", "efsw", "fmt", "frozen"},
```

Comment:
cxxopts is not used by NazaraShader, it's used by nzslc.

Reply:
Removed!